_topic_maps/_topic_map.yml

```diff
@@ -1890,12 +1890,12 @@ Distros: openshift-enterprise,openshift-origin,openshift-online
 Topics:
 - Name: CI/CD overview
   File: index
-# - Name: Builds using Shipwright
-#  Dir: builds_using_shipwright
-#  Distros: openshift-enterprise
-#  Topics:
-#  - Name: Overview of Builds
-#    File: overview-openshift-builds
+- Name: Builds using Shipwright
+  Dir: builds_using_shipwright
+  Distros: openshift-enterprise
+  Topics:
+  - Name: Overview of Builds
```

Comment:
This might be out of scope, but I thought I should mention it. You can choose to address it now or later in another ticket. I am wondering if the "Overview of Builds" module that is nested under the "builds using shipweight" subdirectrory should be renamed. As it stands it doesn't seem very specific to shipwright and there is another subdirectory following named "builds using buildconfig" I am wondering if it could be confusing to see an "overview of builds" title and then "builds using buildconfig" without reading the paragraph for context. I know that is not your intention, but I could see where someone could get confused if they were quickly looking for information.

Reply:
@opayne1, I understand your concern here. But, the content structure you are seeing in the left navigation has already been discussed and approved by PMs, CS, DPM, and other stakeholders. Though we have two flavors for Builds, we do not want to highlight or emphasize much on the technologies behind Builds. One is Shipwright and the other is BuildConfig. The actual product name we have used on the standalone page is just Builds (rather specifying Builds using Shipwright). The intent is to keep both the flavours in just one umbrella, that is Builds. We have already segragted the content at the top level to avoid any confusion. Since the builds using buildconfig content would still be part of the core openshift docs site. With OCP 4.15, plans are to move the buildconfig content as well to our standalone page and then mold the structure in such a way that we do not require any segragation point and the flow of the content would look seamless.

The reason i think it would not be confusing for readers is that when a user clicks the overview of Builds section under the builds using shipwright directory, that person would be redirected to a [separate standalone page](https://docs.openshift.com/builds/1.0/about/overview-openshift-builds.html) like we did for [pipelines](https://docs.openshift.com/pipelines/1.12/about/understanding-openshift-pipelines.html) and [gitops](https://docs.openshift.com/gitops/1.10/understanding_openshift_gitops/about-redhat-openshift-gitops.html). That page covers all the details of Builds using shipwright only. All the details of builds using buildconfig would be maintained on the docs.openshift.com site only.